src/clr_bundle_add.c

```diff
@@ -37,163 +37,65 @@
 
 static char **bundles;
 
-static void print_help(const char *name)
+static void print_help()
```

Comment:
in the other function declaration of print_help you used _void_ as argument and here you are leaving it empty, I would prefer we use one way or the other consistently.